packages/harvester/harvester-cloud-provider/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/harvester/charts/releases/download/harvester-cloud-provider-0.1.8/harvester-cloud-provider-0.1.8.tgz
+url: https://github.com/harvester/charts/releases/download/harvester-cloud-provider-0.1.11/harvester-cloud-provider-0.1.11.tgz
 version: 100.0.1
```

Comment:
Looks like the latest versions we released were `100.0.1+up0.1.9` for csi and `100.0.1+up0.1.8` for cloud provider. You should bump this to `100.0.2` since the upstream reference has been updated.